metrics/ci_integrations/docs/diagrams/ci_integrations_config_validator_interfaces_class_diagram.puml

```diff
@@ -1,101 +1,84 @@
 @startuml ci_integrations_config_validator_class_diagram
```

Comment:
Would it be great to rename this file to be something like `config_validator_base_class_diagram` since it contains not only interfaces now?